config/initializers/sidekiq_alive.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+if defined?(Sidekiq) && defined?(SidekiqAlive)
+  SidekiqAlive.setup do |config|
+    config.callback = proc do
+      Sidekiq::Queue.all.each do |queue|
+        # skip other queues (default, critical, etc.)
+        next unless queue.name.starts_with?(config.queue_prefix.to_s)
+        # keep actual sidekiq-alive-<hostname> queue
+        next if queue.name == "#{config.queue_prefix}-#{SidekiqAlive.hostname}"
+
+        queue.clear
+      end
+    end
+  end
+end
```

Comment:
@sasasasumna  `shutdown_callback` does not work for some reason, so I used `callback` to clean up old sidekiq-alive jobs - when it starts Redis for sure is accessible.